internal/service/alertconfiguration/resource_alert_configuration_test.go

```diff
@@ -338,68 +326,10 @@ func TestAccConfigRSAlertConfiguration_importIncorrectId(t *testing.T) {
 	})
 }
 
-func TestAccConfigRSAlertConfiguration_importConfigNotifications(t *testing.T) {
```

Comment:
Are we losing this specific test? it seems there isn't a counterpart during acc_test (unless I am missing it)

Reply:
it's joined to TestAccConfigRSAlertConfiguration_withNotifications